libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -71,16 +69,17 @@ export class SkyColorpickerInputDirective
    * @deprecated
    */
   @Input()
-  public set initialColor(value: string) {
+  public set initialColor(value: string | undefined) {
     /* istanbul ignore else */
-    if (!this._initialColor && !this.modelValue) {
+    if (!this.#_initialColor && !this.#modelValue) {
       this.writeValue(value);
     }
-    this._initialColor = value;
+
+    this.#_initialColor = value;
   }
 
   public get initialColor(): string {
-    return this._initialColor || SKY_COLORPICKER_DEFAULT_COLOR;
+    return this.#_initialColor || SKY_COLORPICKER_DEFAULT_COLOR;
```

Comment:
Because of the odd logic in the setter, I couldn't move assignment of the default color there.